src/parser/jobs/rdm/Gauge.js

```diff
@@ -301,6 +394,36 @@ export default class Gauge extends Module {
 					<Trans id="rdm.gauge.suggestions.black-mana-lost-why">You lost {this._blackManaLostToImbalance} Black Mana due to overage of White Mana</Trans>
 				</Fragment>,
 			}))
+
+			this.suggestions.add(new TieredSuggestion({
```

Comment:
Duping up a bit on suggestions - these are basically saying the same thing from the opposite side of the mana gauge. Consider consolidating them into a single metric.